projects/packages/woocommerce-analytics/src/class-woo-analytics-trait.php

```diff
@@ -434,15 +433,15 @@ public function process_event_properties( $event_name, $properties = array(), $p
 			)
 		);
 
+		$js = "{'_en': '" . esc_js( $event_name ) . "'";
+
 		// ch param is needed to identify ClickHouse queries in the JS Analytics library.
 		if ( $this->is_clickhouse( $event_name ) ) {
-			$js = "{'_en': '" . esc_js( $this->get_clickhouse_event( $event_name ) ) . "','ch':'1'";
-		} else {
-			$js = "{'_en': '" . esc_js( $event_name ) . "'";
+			$js .= "','ch':'1'";
```

Comment:
There an extra single quote `'` here that causes the syntax error:

```
_wca.push({'_en': 'woocommerceanalytics_session_started'','ch':'1','session_id': .....`
```

In this example there's an extra `'` after `woocommerceanalytics_session_started`.